lib/belugas/ruby/parser/gemfile.rb

```diff
@@ -29,11 +29,21 @@ def dependencies
         end
 
         def ruby_version
-          ENV['FALLBACK_RUBY_VERSION'] || content.scan(/ruby ["'](.*?)["']/).flatten[0] || FALLBACK_RUBY_VERSION
+          ENV['FALLBACK_RUBY_VERSION'] || remove_symbols_from_ruby_version(content.scan(/ruby ["'](.*?)["']/).flatten[0]) || FALLBACK_RUBY_VERSION
         end
 
         private
 
+        def remove_symbols_from_ruby_version(ruby_version)
+          return nil if ruby_version.nil?
+
+          ruby_version.delete!("<")
+          ruby_version.delete!(">")
+          ruby_version.delete!("=")
+          ruby_version.delete!(' ')
```

Comment:
Consider using a regex instead of `.delete` method, it'll be faster and a one liner, something like this should work:


```
def remove_symbols_from_ruby_version(ruby_version)
  return nil if ruby_version.nil?
  DIGITS_REGEX = /(\d+[,.]\d+)/
  ruby_version.match DIGITS_REGEX[0].to_s
end
```